mappings/net/minecraft/util/StreamHandler.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS yy net/minecraft/util/StreamHandler
```

Comment:
This is a class that allows a `Stream` to be iterated more than once. `StreamHandler` is not a descriptive name. I propose `CyclingStream` or `LoopingStream`.

Reply:
Oh good idea didn't notice that part